diff -urN rustc-1.35.0-src.old/src/bootstrap/sanity.rs rustc-1.35.0-src/src/bootstrap/sanity.rs
--- rustc-1.35.0-src.old/src/bootstrap/sanity.rs	2019-06-03 03:47:46.202107101 -0000
+++ rustc-1.35.0-src/src/bootstrap/sanity.rs	2019-06-03 03:48:57.632108423 -0000
@@ -186,10 +186,10 @@
                         panic!("couldn't find libc.a in musl dir: {}",
                                root.join("lib").display());
                     }
-                    if fs::metadata(root.join("lib/libunwind.a")).is_err() {
-                        panic!("couldn't find libunwind.a in musl dir: {}",
-                               root.join("lib").display());
-                    }
+                    //if fs::metadata(root.join("lib/libunwind.a")).is_err() {
+                    //    panic!("couldn't find libunwind.a in musl dir: {}",
+                    //           root.join("lib").display());
+                    //}
                 }
                 None => {
                     panic!("when targeting MUSL either the rust.musl-root \
