diff -urN old/swaylock/main.c new/swaylock/main.c
--- old/swaylock/main.c	2018-11-29 08:10:00.000000000 -0000
+++ new/swaylock/main.c	2019-01-27 23:37:08.026321848 -0000
@@ -4,6 +4,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <getopt.h>
+#include <poll.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -961,7 +962,7 @@
 	}
 
 	state.eventloop = loop_create();
-	loop_add_fd(state.eventloop, wl_display_get_fd(state.display), POLL_IN,
+	loop_add_fd(state.eventloop, wl_display_get_fd(state.display), POLLIN,
 			display_in, NULL);
 
 	state.run_display = true;
